react-design/my-app/src/NextButton.tsx

```diff
@@ -0,0 +1,9 @@
+import './NextButton.css';
+
+export default function NextButton() {
+  return (
+    <div id="next-button">
```

Comment:
I'm curious why you don't just use `<button>`. For accessibility purposes, it's best to use the DOM tag that most closely represents the purpose.